crates/relayer/src/chain/cosmos.rs

```diff
@@ -1078,9 +1078,9 @@ impl ChainEndpoint for CosmosSdkChain {
         ChainConfig::CosmosSdk(self.config.clone())
     }
 
-    fn ibc_version(&self) -> Result<Option<semver::Version>, Error> {
+    fn version_specs(&self) -> Result<(Option<semver::Version>, semver::Version), Error> {
```

Comment:
Here I meant to suggest both renaming the method and returning the full VersionSpecs struct instead of a tuple.

Reply:
Oh my bad, should I have it return the `Specs` struct https://github.com/informalsystems/hermes/blob/master/crates/relayer/src/chain/cosmos/version.rs#L37. Or would it be better to have a struct with all `Option<>` to avoid issues with non-SDK chains? E.g.:

```
pub struct VersionSpecs {
    pub cosmos_sdk: Option<semver::Version>,
    pub ibc_go: Option<semver::Version>,
    pub tendermint: Option<semver::Version>,
    pub comet: Option<semver::Version>,
}
```

Reply:
Oh right, yeah that's the way to go!